web/src/helpers/map.js

```diff
@@ -1,6 +1,6 @@
-import { max, min, mean } from 'lodash';
+import { max, min, mean, isArray } from 'lodash';
 
-export function getCenteredLocationViewport([longitude, latitude]) {
+export function getCenteredLocationViewport([longitude, latitude]) {''
```

Comment:
This looks a bit funky... `''`

Reply:
Why is the linter not catching this?

Reply:
Dunno, but thanks for catching it! Fixed now